.github/workflows/test-network-basic.yaml

```diff
@@ -0,0 +1,51 @@
+name: Test Network Basic
+run-name: ${{ github.actor }} is running the test-network basic tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - release-1.4
+      - release-2.2
+  pull_request:
+    branches:
+      - main
+      - release-1.4
+      - release-2.2
+
+jobs:
+  basic:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        chaincode-language:
+          - go
+          - javascript
+          - typescript
+          - java
+
+    steps:
+      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
+
+      - name: checkout
+        uses: actions/checkout@v3
+
+      - name: Install fabric CLI
+        run: |
+          curl -sSL https://raw.githubusercontent.com/hyperledger/fabric/main/scripts/install-fabric.sh | bash -s -- binary
+          echo ${PWD}/bin >> $GITHUB_PATH
+
+      - name: Install retry CLI
+        run: curl -sSL https://raw.githubusercontent.com/kadwanev/retry/master/retry -o ./bin/retry && chmod +x ./bin/retry
+
+      - name: Pull Fabric Docker Images
+        run: ./ci/scripts/pullFabricImages.sh
```

Comment:
any reason for a different script - `install-fabric.sh` can also do docker images?

Reply:
The docker image pulls are not right - was going to hit this in another PR but it can go in here as well.